### PR TITLE
docs(common): Remove promises for the future

### PR DESCRIPTION
**Pull request by dimodi**

### CODE REVIEW

File: components/textarea/overview.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The Blazor TextArea provides various parameters to configure the component:
 | ----------- | ----------- | ----------- |
 | `AutoCapitalize` | `string` | A `string` that maps to the [`autocapitalize`](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autocapitalize) attribute of the HTML element. It's applicable only for touch devices and virtual keyboards. |
 | `AutoComplete` | `bool` | Maps to the autocomplete attribute of the HTML `<textarea>`. |
-| `AutoSize` | `bool` | Specifies if the TextArea will adjust its height based on the user input. You can [use CSS to limit the resizing up to a max height]({%slug textarea-kb-autosize-max-height%}). This parameter will be deprecated in a future version. Use `ResizeMode` with `Auto` value instead. |
+| `AutoSize` | `bool` | Specifies if the TextArea will adjust its height based on the user input. You can [use CSS to limit the resizing up to a max height]({%slug textarea-kb-autosize-max-height%}). This parameter is deprecated. Use `ResizeMode` with `Auto` value instead. |
```
Comment (ikoevska) [inline components/textarea/overview.md:58]: The fact that this is deprecated is important and should be made more prominent. You can do this by placing the respective information first (this way, people can decide if they want to read more about it or not) or by separating it on a new line using a line breaker.

**Review by yordan-mitev — APPROVED** [review]

File: knowledge-base/form-chrome-autofill.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ The application is not given the chance to respond to the autofill action of the
 
 1. Run this snippet in the browser
 2. Fill in the form and click Submit
-3. In the browser popup asking you to save the data for the future, confirm and let Chrome save it
+3. In the browser popup asking you to save the form data, confirm and let Chrome save it
```
Comment (ikoevska) [inline knowledge-base/form-chrome-autofill.md:45]: Out of curiosity, why isn't there any ending punctuation throughout the list?

File: knowledge-base/requires-valueexpression.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ The most common reason for this error is a combination of the following:
...
+This error comes from the framework and applies to generic inputs as well, not only the Telerik components. At the time of writing, I was not able to find an official resource for the `ValueExpression` feature. The closest is the example for a `ValidationMessage` at [https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-3.1#validation-summary-and-validation-message-components](https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-3.1#validation-summary-and-validation-message-components).
```
Comment (ikoevska) [inline knowledge-base/requires-valueexpression.md:39]:
```suggestion
This error comes from the framework and applies to generic inputs as well, not only the Telerik components. At the time of writing, an official resource for the `ValueExpression` feature was not readily available. The closest was the example for a `ValidationMessage` at [https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-3.1#validation-summary-and-validation-message-components](https://docs.microsoft.com/en-us/aspnet/core/blazor/forms-validation?view=aspnetcore-3.1#validation-summary-and-validation-message-components).
```

File: upgrade/breaking-changes/3-0-0.md [hunk 4]
```diff
@@ -486,7 +486,7 @@ public class TreeItem
     </tbody>
 </table>
 
-- **Window** - changed `Size` parameter type from `WindowSize` enum to `string`. There is a new static class `Telerik.Blazor.ThemeConstants.Window.Size` with a predefined set of size properties. The `Size` property will have effect in future UI for Blazor versions. To maintain the old behavior, set the Window `Width` to `300px`, `800px` or `1200px`.
+- **Window** - changed `Size` parameter type from `WindowSize` enum to `string`. There is a new static class `Telerik.Blazor.ThemeConstants.Window.Size` with a predefined set of size properties. To maintain the old behavior, set the Window `Width` to `300px`, `800px` or `1200px`.
```
Comment (ikoevska) [inline upgrade/breaking-changes/3-0-0.md:489]:
```suggestion
- **Window**&mdash;changed `Size` parameter type from `WindowSize` enum to `string`. There is a new static class `Telerik.Blazor.ThemeConstants.Window.Size` with a predefined set of size properties. To maintain the old behavior, set the Window `Width` to `300px`, `800px` or `1200px`.
```

File: upgrade/overview.md [hunk 5]
```diff
@@ -99,7 +99,7 @@ While using a trial license, a watermark will be rendered over the components an
 
 If you have a commercial license, but you still see the trial watermark and banner, especially after publishing an app to a staging or live server, your build has used the trial `Telerik.Blazor.dll` assembly.
 
-The most common reason for the problem is that the framework decides to incorrectly copy a trial version of our assembly into the `dist` folder of the app, even though it puts the correct licensed version at the root of the publish folder. This seems like a problem in the build process of the framework, and hopefully it will get fixed in future versions.
+The most common reason for the problem is that the framework decides to incorrectly copy a trial version of our assembly into the `dist` folder of the app, even though it puts the correct licensed version at the root of the publish folder. This seems like a problem in the build process of the framework.
```
Comment (ikoevska) [inline upgrade/overview.md:102]:
```suggestion
The most common reason for the problem is that the framework decides to incorrectly copy a trial version of the Telerik UI for Blazor assembly into the `dist` folder of the app, even though it puts the correct licensed version at the root of the publish folder. This seems like a problem in the build process of the framework.
```

**Review by ikoevska — APPROVED** [review]

File: knowledge-base/form-chrome-autofill.md (hunk 2 above)
Comment (dimodi) [reply knowledge-base/form-chrome-autofill.md]: Oversight.

**Pull request merged**